dspace-api/src/main/java/org/dspace/discovery/IndexClient.java

```diff
@@ -150,10 +168,55 @@ public static void main(String[] args) throws SQLException, IOException, SearchS
             indexer.optimize();
         } else if (line.hasOption('s')) {
             checkRebuildSpellCheck(line, indexer);
+        } else if (line.hasOption("item_uuid")) {
+            String itemUUID = line.getOptionValue("item_uuid");
+            Item item = ContentServiceFactory.getInstance().getItemService().find(context, UUID.fromString(itemUUID));
+            indexer.indexContent(context, item, line.hasOption("f"));
```

Comment:
Best to add a check to see that the item was found, if it wasn't found a NullPointer will follow.

Reply:
done, see https://github.com/4Science/DSpace/pull/65/commits/8de7a50f5be1d2f85e37f8659decb772b5780ee1
same behavior than the existing -i option